### PR TITLE
Portfolio: application deploy (branch: f6-deployment-gh).

### PR DESCRIPTION
**Pull request by jcmunav63**
**Description**

For this milestone, you will make your website available online.

IMPORTANT NOTE: Read all requirements before you start building your project.

**General requirements**

- Make sure that there are [no linter errors](https://github.com/microverseinc/linters-config).
- Make sure that you used the correct flow ([Github Flow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md).
- Make sure that you documented your work [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md).

**Project requirements**

IMPORTANT NOTE: For each Microverse project, you need to create a separate pull request. [Read how to work with multiple pull requests in your one repository in the most efficient way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/multiple_pull_requests.md).

- Deploy your website using GitHub Pages.
- Check the online version of your portfolio and make sure that the page works properly.
- Update the README of your repository to include a link to the online version.
- If by any chance you have deployed your app earlier, open an issue with a link to the online version of your app. Otherwise, submit a link to the PR with the changes required for this deployment.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,16 +22,16 @@
...
-**Portfolio: desktop version** A project for finishing the desktop portfolio app!
+**Portfolio: Deployment of desktop version** A project for deployment of mobile & desktop portfolio app!
 
 ## 🛠️ Built With <a name="built-with">
 ### Tech Stack <a name="tech-stack"></a>
 - HTML
-- CSS
+- CSS & Media Queries
 
 </a>
```
Comment (tresorsawasawa) [inline README.md]: - [ ] Good job by providing a working deployed link of your project. However as required for the project, kindly ensure that your README is updated to include a link to the online version. That means you should add the `Live section` you removed earlier that include the deployed link.
```
<!-- LIVE DEMO -->

## 🚀 Live Demo <a name="live-demo"></a>

> Add a link to your deployed project.

- [Live Demo Link](https://jcmunav63.github.io/mv_portfolio1/)
```
For a short remind where you should put the `Live Demo` section, just take a look again on the **[template](https://github.com/microverseinc/readme-template)**

**Review by tresorsawasawa — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @jcmunav63 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Follow GitHub flow :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (jcmunav63) [reply README.md]: Dear reviewer @tresorsawasawa:

I included the live demo link that you request in the README.md file, under the section **"Deployment (Github)"**. I don't know if this is enough to get this project approved. Thank you.

**Review by emmiiorji — APPROVED** [review]
Hi @jcmunav63, 

# Status: Approved ✅

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

<div align='center'>

</div>


## Good Highlights✔️

- Your project is live

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**